src/main/steps/edge-case/upload-other-information/uploadDocPostController.ts

```diff
@@ -0,0 +1,339 @@
+  SIZE_ERROR?: string;
+  TOTAL_FILES_EXCEED_ERROR?: string;
+  CONTINUE_WITHOUT_UPLOAD_ERROR?: string;
+  NO_FILE_UPLOAD_ERROR?: string;
+};
+
+export const FIS_COS_API_URL: URL_OF_FILE = config.get(FIS_COS_API_BASE_URL);
+
+/**
+ * @FileHandler
+ */
+export const FileMimeType: Partial<Record<keyof FileType, keyof FileMimeTypeInfo>> = {
+  doc: 'application/msword',
+  docx: 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+  pdf: 'application/pdf',
+  png: 'image/png',
+  xls: 'application/vnd.ms-excel',
+  xlsx: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+  jpg: 'image/jpeg',
+  txt: 'text/plain',
+  rtf: 'application/rtf',
+  rtf2: 'text/rtf',
+  gif: 'image/gif',
+  mp4audio: 'audio/mp4',
+  mp4video: 'video/mp4',
+};
+
+export class FileValidations {
+  /**
+   *
+   * @param req
+   * @returns
+   */
+
+  static ResourceReaderContents = (req: AppRequest<AnyObject>): FileUploadErrorTranslatables => {
+    let SystemContent: any | FileUploadErrorTranslatables = {};
+    const SystemLangauge = req.session['lang'];
+    const resourceLoader = new ResourceReader();
+    resourceLoader.Loader('upload-appeal-form');
+    const ErrorInLangauges = resourceLoader.getFileContents().errors;
+    switch (SystemLangauge) {
+      case 'en':
+        SystemContent = ErrorInLangauges.en;
+        break;
+      case 'cy':
+        SystemContent = ErrorInLangauges.cy;
+        break;
+      default:
+        SystemContent = ErrorInLangauges.en;
+    }
+    return SystemContent;
+  };
+
+  /**
+   *
+   * @param fileSize
+   * @returns
+   */
+  static sizeValidation = (mimeType: string, fileSize: number): boolean => {
+    const KbsInMBS = mimeType.endsWith('/mp4')
+      ? Number(config.get('documentUpload.validation.multimediaSizeInKB'))
+      : Number(config.get('documentUpload.validation.sizeInKB'));
+    if (fileSize <= KbsInMBS) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  /**
+   *
+   * @param mimeType
+   * @returns
+   */
+  static formatValidation = (mimeType: string): boolean => {
+    const allMimeTypes = Object.values(FileMimeType);
+    const checkForFileMimeType = allMimeTypes.filter(aMimeType => aMimeType === mimeType);
+    return checkForFileMimeType.length > 0;
+  };
+}
+
+@autobind
+export default class UploadDocumentController extends PostController<AnyObject> {
+  constructor(protected readonly fields: FormFields | FormFieldsFn) {
+    super(fields);
+  }
+
+  async PostDocumentUploader(req: AppRequest<AnyObject>, res: Response): Promise<void> {
+    if (req.session.hasOwnProperty('otherCaseInformation')) {
+      const TotalUploadDocuments = req.session.otherCaseInformation.length;
+
+      if (TotalUploadDocuments === 0) {
+        const errorMessage = FileValidations.ResourceReaderContents(req).CONTINUE_WITHOUT_UPLOAD_ERROR;
+        this.uploadFileError(req, res, errorMessage);
+      } else {
+        // const CaseId = req.session.userCase['id'];
```

Comment:
Large block of commented code. Can we safely delete this?

Reply:
@osheaf01 This code (copied from the existing document upload screens from the repo we have forked) is there to update the case that will ultimately be created earlier in the journey, but since we don't yet have the ability to create a case in the CCD data store, we can't update it.  I have left the code there, but commented out, as we will need to update the case here once case creation is working.

Reply:
OK that works for me.